lib/start_bootstrap_admin_theme/engine.rb

```diff
@@ -1,5 +1,15 @@
 module StartBootstrapAdminTheme
   class Engine < ::Rails::Engine
     isolate_namespace StartBootstrapAdminTheme
+
+    initializer 'start_bootstrap_admin_theme.assets.precompile' do |app|
+      asset_paths = Pathname.new(__FILE__).join('../../../vendor/assets').glob('**/*.*').map do |path|
+        path.to_s
+          .sub(%r{\A.*/vendor/assets/\w+/}, '')
+          .sub(/\.erb/, '')
+      end
+
+      app.config.assets.precompile += asset_paths
```

Comment:
When the gem is added to a Rails app, and you run `rails assets:precompile`, this config allows the gem's assets to be precompiled as separate assets from the app. See https://guides.rubyonrails.org/engines.html#separate-assets-precompiling for more info.